nips2016_NAMPI/process/officialReviewProcess.js

```diff
@@ -0,0 +1,31 @@
+function(){
+    var or3client = lib.or3client;
+    
+    var origNote = or3client.or3request(or3client.notesUrl+'?id='+note.forum, {}, 'GET', token);
+    var list = note.invitation.replace(/_/g,' ').split('/');
```

Comment:
This piece of code is present in all the process functions, could be move it to the or3client? as a helper for the script developer.